### DISCUSSION TITLE
TypeError: MaskAnnotator.__init__() got an unexpected keyword argument 'color_map'

### DISCUSSION THREAD

**Discussion by aimanmutasem** [roboflow/supervision#515] [Q&A] [answered]
### Search before asking

- [X] I have searched the Supervision [issues](https://github.com/roboflow/supervision/issues) and found no similar bug report.

### Bug

Hello, thank you for the grate repo. I'm following the tutorial: https://blog.roboflow.com/how-to-use-segment-anything-model-sam/.  The problem is when I want to draw masks on an image using provided detections with Segment-Anything (SAM).

The error is:
`
TypeError                                 Traceback (most recent call last)
[<ipython-input-28-b3e7c96dd9c3>](https://localhost:8080/#) in <cell line: 1>()
----> 1 mask_annotator = sv.MaskAnnotator(color_map = "index")
      2 
      3 detections = sv.Detections.from_sam(sam_result)
      4 
      5 annotated_image = mask_annotator.annotate(scene=image_bgr.copy(), detections=detections)

TypeError: MaskAnnotator.__init__() got an unexpected keyword argument 'color_map'
`

### Minimal Reproducible Example

mask_annotator = sv.MaskAnnotator(color_map = "index") 

detections = sv.Detections.from_sam(sam_result)

annotated_image = mask_annotator.annotate(scene=image_bgr.copy(), detections=detections)

sv.plot_images_grid(
    images=[image_bgr, annotated_image],
    grid_size=(1, 2),
    titles=['source image', 'segmented image']
)

### Are you willing to submit a PR?

- [x] Yes I'd like to help by submitting a PR!

**Comment by onuralpszr** [comment, marked as answer]
Hello, I moved issue to Q&A because it is not a bug and color_map become color_lookup and example use case

```python
mask_annotator = sv.MaskAnnotator(color_lookup = sv.ColorLookup.INDEX) # INDEX # CLASS #TRACK
```
Enum is here : https://github.com/roboflow/supervision/blob/develop/supervision/annotators/utils.py#L11

**Reply by aimanmutasem** [reply]
Thank you @onuralpszr,  mask_annotator = sv.MaskAnnotator(color_lookup = sv.ColorLookup.INDEX) # INDEX # CLASS #TRACK works with me..

**Reply by onuralpszr** [reply]
@aimanmutasem  please mark as an answer :)
